src/main/kotlin/core/service/PokemonService.kt

```diff
@@ -0,0 +1,59 @@
+package core.service
+
+import core.model.Pokemon
+import core.model.User
+import core.model.UserPokemon
+import core.model.UserToPokemon
+import org.springframework.data.jdbc.repository.query.Query
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+import org.springframework.stereotype.Service
+import java.net.URL
+
+@Repository
+interface PokemonRepository : CrudRepository<Pokemon, String> {
+    @Query("select * from pokemon")
```

Comment:
Instead of hardcoding the queries in the service, we should be utilizing the ORM to do the work of filtering for us. https://www.baeldung.com/java-dao-vs-repository